enterprise/server/remote_execution/uffd/uffd.go

```diff
@@ -125,6 +125,9 @@ func (h *Handler) Start(ctx context.Context, socketPath string, memoryStore *cop
 
 	go func() {
 		h.handleErr = h.handle(ctx, memoryStore)
+		if h.handleErr != nil {
+			log.CtxErrorf(ctx, "UFFD handler failed: %s", h.handleErr)
+		}
```

Comment:
Added this logging back in case we forget to check `context.Cause()` somewhere where we should be.